### ISSUE TITLE
no_std examples fails after `usbd-hid` update

### ISSUE THREAD

**Issue by eivindbergem** [rustne-kretser/noline#28]
```
error[E0277]: the trait bound `usbd_hid_descriptors::MainItemKind: From<std::string::String>` is not satisfied
   --> /home/eivind.bergem/.cargo/registry/src/index.crates.io-6f17d22bba15001f/usbd-hid-macros-0.6.0/src/spec.rs:512:43
    |
512 |             self.set_item(name, item_kind.into(), settings, bits, quirks);
    |                                           ^^^^ the trait `From<std::string::String>` is not implemented for `usbd_hid_descriptors::MainItemKind`, which is required by `std::string::String: Into<_>`
    |
    = help: the trait `From<&str>` is implemented for `usbd_hid_descriptors::MainItemKind`
    = help: for that trait implementation, expected `&str`, found `std::string::String`
    = note: required for `std::string::String` to implement `Into<usbd_hid_descriptors::MainItemKind>`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `usbd-hid-macros` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...

```

See https://github.com/twitchyliquid64/usbd-hid/issues/72 for more info.

**Comment by eivindbergem** [comment]
We should check in lock files for the examples, as this would have prevented the error.

**Comment by eivindbergem** [comment]
The `rp2040` example can be fixed by updating to `usbd-hid` ` 0.8.2`.
